Use of tainted arguments in exec is deprecated at /var/www/cgi-bin/cvsweb.cgi line 2043.
===================================================================
RCS file: /home2/webcvs/mirror/kdeedu/kstars/kstars/indi/webcam/port.cpp,v
retrieving revision 1.3
retrieving revision 1.3.2.1
diff -u -p -r1.3 -r1.3.2.1
--- kdeedu/kstars/kstars/indi/webcam/port.cpp	2004/07/06 21:19:01	1.3
+++ kdeedu/kstars/kstars/indi/webcam/port.cpp	2004/08/27 19:33:34	1.3.2.1
@@ -38,7 +38,7 @@
 #endif /* LOCKING */
 
 #ifdef __linux__
-  #if defined(arm) || defined(__hppa__)
+  #if defined(arm) || defined(__hppa__) || defined(__sparc__) || defined(__ppc__) || defined(__powerpc__)
   #include <fcntl.h>
   #else
   #include <sys/io.h>
@@ -74,7 +74,7 @@ port_t::port_t(int iport) {
 #endif /* LOCKING */
 
 #ifdef LINUX
-#if defined(arm) || defined(__hppa__)
+#if defined(arm) || defined(__hppa__) || defined(__sparc__) || defined(__ppc__) || defined(__powerpc__)
   if ((devport = open("/dev/port", O_RDWR)) < 0) {
     perror("open /dev/port");
     return;
@@ -118,7 +118,7 @@ port_t::~port_t(void) {
   unlock(port);
 #endif /* LOCKING */
 #ifdef LINUX
-#if defined(arm) || defined(__hppa__)
+#if defined(arm) || defined(__hppa__) || defined(__sparc__) || defined(__ppc__) || defined(__powerpc__)
   if (devport >= 0)
     close(devport);
 #else
Use of tainted arguments in exec is deprecated at /var/www/cgi-bin/cvsweb.cgi line 2043.
===================================================================
RCS file: /home2/webcvs/mirror/kdeedu/kstars/kstars/indi/webcam/port.h,v
retrieving revision 1.3
retrieving revision 1.3.2.1
diff -u -p -r1.3 -r1.3.2.1
--- kdeedu/kstars/kstars/indi/webcam/port.h	2004/07/06 21:19:01	1.3
+++ kdeedu/kstars/kstars/indi/webcam/port.h	2004/08/27 19:33:34	1.3.2.1
@@ -35,7 +35,7 @@
 #include <unistd.h>
 
 #ifdef __linux__
-  #if !defined(arm) && !defined(__hppa__)
+  #if !defined(arm) && !defined(__hppa__) && !defined(__sparc__) && !defined(__ppc__) && !defined(__powerpc__)
   #include <sys/io.h>
   #endif /* !arm */
 #elif defined(QNX)
@@ -55,7 +55,7 @@
 #error Please define a platform in the Makefile
 #endif
 
-#if defined(arm) || defined(__hppa__)
+#if defined(arm) || defined(__hppa__) || defined(__sparc__) || defined(__ppc__) || defined(__powerpc__)
 static char ports_temp;
 
 #ifdef inb
@@ -114,7 +114,7 @@ private:
 #ifdef FREEBSD
   FILE *devio;
 #endif
-#if defined(__linux__) && (defined(arm) || defined(__hppa__))
+#if defined(__linux__) && (defined(arm) || defined(__hppa__) || defined(__sparc__) || defined(__ppc__) || defined(__powerpc__))
   int devport;
 #endif
 };
